### PR TITLE
Bench 152 write unit tests for LineItemSerializer, CategorySerializer and CategoryDetailSerializer

### PR DESCRIPTION
**Pull request by JoeCSykes**

### CODE REVIEW

File: answerking_app/models/models.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from decimal import Decimal
 
 from django.db import models
+from django.db.models import QuerySet
```
Comment (NogaAD) [inline answerking_app/models/models.py]: Do you use this?

File: answerking_app/tests/test_unit/test_unit_model.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+from decimal import Decimal
+
+from django.test import TestCase
+
+from answerking_app.models.models import Order, Product, LineItem
+from answerking_app.tests.BaseTestClass import TestBase
+from answerking_app.tests.test_unit.UnitTestBaseClass import UnitTestBase
+
```
Comment (NogaAD) [inline answerking_app/tests/test_unit/test_unit_model.py]: Remove TestCase and testBase?

**Review by MichaelOSullivanAnswer — APPROVED** [review]

File: answerking_app/tests/test_unit/test_unit_category_serializer.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
+import datetime
```
Comment (NogaAD) [inline answerking_app/tests/test_unit/test_unit_category_serializer.py]: remove this

File: answerking_app/tests/test_unit/UnitTestBaseClass.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timedelta
```
Comment (NogaAD) [inline answerking_app/tests/test_unit/UnitTestBaseClass.py]: Also this is unnecessary

File: answerking_app/models/models.py (hunk 1 above)
Comment (JoeCSykes) [reply answerking_app/models/models.py]: got rid of

File: answerking_app/tests/test_unit/test_unit_model.py (hunk 2 above)
Comment (JoeCSykes) [reply answerking_app/tests/test_unit/test_unit_model.py]: got rid of

File: answerking_app/tests/test_unit/test_unit_category_serializer.py (hunk 3 above)
Comment (JoeCSykes) [reply answerking_app/tests/test_unit/test_unit_category_serializer.py]: removed

File: answerking_app/tests/test_unit/UnitTestBaseClass.py (hunk 4 above)
Comment (JoeCSykes) [reply answerking_app/tests/test_unit/UnitTestBaseClass.py]: removed

**Review by NogaAD — APPROVED** [review]

**Pull request merged**